### PR TITLE
CH-1156 Add toggle for viewing global gateways

### PR DESCRIPTION
**Pull request by vicmgs**

### CODE REVIEW

**Comment by allenan** [comment]
is there a way to see global hotspots when I don't have any?

File: app/components/gateway/GatewayIndexList.js [hunk 1]
```diff
@@ -155,12 +181,31 @@ class GatewayIndexList extends Component {
...
+            <div className={classes.checkbox}>
+              <Checkbox
+                checked={showGlobalGateways}
+                onChange={this.handleMapChange}
+                value="showGlobalGateways"
+                color="primary"
+                disableRipple
+              />
+              <Typography variant="display4" className={classes.label}>
+                Show Global Gateways
```
Comment (allenan) [inline app/components/gateway/GatewayIndexList.js]: s/gateways/hotspots

**Comment by allenan** [comment]
image
we should make sure that the popover is on top of all cluster markers

**Comment by vicmgs** [comment]
Coco wanted the hotspots map to be unavailable until the user has a gateway registered

**Comment by allenan** [comment]
ok that's fine I guess, though I think maybe there should be some tertiary action on the blank slate screen that lets you see global hotspots, but that can be a separate discussion

**Comment by allenan** [comment]
I think the names on the side might clutter up the view in certain cases, but we can leave it for now and see if anyone complains

image

**Comment by vicmgs** [comment]
yea, the map is useful for non hotspot owners as well. Also, clubhouse ticket 1448 is actually more complicated than it seems to fix and changing how the empty state design looks can probably get rid of that ticket.

**Comment by allenan** [comment]
interesting, good to know

**Pull request merged**